### PR TITLE
Built in 3DS verification support

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

File: sdk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -14,4 +14,5 @@
     <string name="label_expiry_date">Expiry date</string>
     <string name="label_security_code">Security code</string>
     <string name="menu_card_io">Scan from Camera</string>
+    <string name="verify_3ds_title">Verify 3DS</string>
```
Comment (chakrit) [inline sdk/src/main/res/values/strings.xml]: should be `title_verify_3ds`

File: sdk/build.gradle [hunk 2]
```diff
@@ -80,6 +80,8 @@ uploadArchives {
 dependencies {
     compile 'joda-time:joda-time:2.9.2'
     compile 'com.squareup.okhttp3:okhttp:3.2.0'
-    provided 'io.card:android-sdk:5.4.0'
+    compile 'com.android.support:support-v4:24.2.1'
+    compile 'com.android.support:appcompat-v7:24.2.1'
```
Comment (chakrit) [inline sdk/build.gradle]: ☝️ need to get rid of these.

File: app/src/main/java/co/omise/android/example/MainActivity.java [hunk 3]
```diff
@@ -28,4 +33,30 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
...
+            intent.putExtra(Verify3DSActivity.EXTRA_AUTHORIZED_URL, "https://api.omise.co/payments/paym_12345/authorize");
+            startActivityForResult(intent, 0x3d5);
+            return true;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        if (requestCode == 0x3d5 && resultCode == RESULT_OK) {
```
Comment (chakrit) [inline app/src/main/java/co/omise/android/example/MainActivity.java]: where does `0x3d5`  come from a?

Comment (pitiphong-p) [inline app/src/main/java/co/omise/android/example/MainActivity.java]: From `3DS`

**Pull request merged**